iree/compiler/Codegen/SPIRV/test/tile_and_vectorize.mlir

```diff
@@ -26,11 +26,7 @@ hal.executable private @matmul  {
       workgroup_size = [16: index, 8: index, 1: index],
       translation.info = {passPipeline = 6 : i32, workloadPerWorkgroup = [8, 16]}
     }
-    builtin.module attributes {
-      spv.target_env =
-        #spv.target_env<#spv.vce<v1.3, [Shader], [SPV_KHR_storage_buffer_storage_class]>,
-                        {max_compute_workgroup_invocations = 128 : i32,
-                         max_compute_workgroup_size = dense<[128, 128, 64]> : vector<3xi32>}>} {
+    builtin.module {
```

Comment:
are these attributes okay to remove from the test?

Reply:
This is fine. They are moved to `hal.executable.variant` op.

Reply:
Yes. We right now don't rely on such information anymore after the pipeline refactoring. Actually it's a good thing to clean them up. Let me do a pass afterwards.

Reply:
As more context, now the pipeline attaches the configuration as the `#config` at L11 to drive CodeGen. `#config` is typically attached by an earlier pass. But here this test does not involve that pass (`SPIRVLowerExecutableTargetPass`).